src/main/java/frc/robot/subsystems/drive/SwerveModuleIO_Real.java

```diff
@@ -74,18 +73,14 @@ public SwerveModuleIO_Real(int drivingCANId, int turningCANId, double chassisAng
       // Invert the turning encoder, since the output shaft rotates in the opposite
       // direction of
       // the steering motor in the MAXSwerve Module.
-      m_turningEncoder.setInverted(SwerveModuleConstants.kTurningEncoderInverted);
-      m_drivingSparkMax.setInverted(SwerveModuleConstants.kDrivingEncoderInverted);
 
       errors += check(m_turningEncoder.setInverted(SwerveModuleConstants.kTurningEncoderInverted));
+      m_drivingSparkMax.setInverted(SwerveModuleConstants.kDrivingEncoderInverted);
```

Comment:
this should probably be one of the errors += check lines

Reply:
https://codedocs.revrobotics.com/java/com/revrobotics/cansparkbase#setInverted(boolean)
https://codedocs.revrobotics.com/java/com/revrobotics/absoluteencoder
see documentation - it actually does NOT allow for this to be added to the errors line because it's a void method as opposed to a RevLibError